.github/fetch_release.py

```diff
@@ -0,0 +1,49 @@
+from os import environ as env
+import requests
+import time
+import json
+
+if 'RELEASE' in env and env['RELEASE']:
+    release = env['RELEASE'].strip()
+else:
+    release = "latest"
+
+release_arg = "latest" if release == "latest" else f"tags/{release}"
+
+print(f"release_arg set to {release_arg}")
+
+release_url = f"https://api.github.com/repos/microsoft/git-credential-manager-core/releases/{release_arg}"

```

Comment:
I know access to the GitHub API can be throttled when used unauthenticated. Using the `GITHUB_TOKEN` might be useful, but not required for this PR.